tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{26,27,32,33,34}
+envlist = py{27,32,33,34,35}
```

Comment:
Okay cool, I guess it's because of Reportlab judging from your previous build: https://travis-ci.org/xhtml2pdf/xhtml2pdf/jobs/97829849